components/AddProjectForm/index.js

```diff
@@ -0,0 +1,98 @@
+import styled from "styled-components";
+import { StyledButton } from "../styles";
+
+export default function AddProjectForm({
+  onCancel,
+  handleAddProjectSubmit,
+  handleChangeProjectStatus,
+  handleChangeProjectFeeling,
+}) {
+  return (
+    <>
+      <ProjectForm onSubmit={(e) => handleAddProjectSubmit(e)}>
+        <RowSection>
+          <label htmlFor="status">status</label>
+          <select id="status" onChange={(e) => handleChangeProjectStatus(e)}>
+            <option>--status--</option>
+            <option value="planning">Planning</option>
+            <option value="active">Active</option>
+            <option value="completed">Completed</option>
+            <option value="hibernating">Hibernating</option>
+          </select>
+          <label htmlFor="happiness">Feeling</label>
+          <select id="feeling" onChange={(e) => handleChangeProjectFeeling(e)}>
+            <option>--feeling--</option>
```

Comment:
If this label belongs to the `feeling` select, you should change its `htmlFor` to "feeling"